src/App.jsx

```diff
@@ -1,13 +1,28 @@
-import { Link } from "react-router-dom";
+import React from 'react'
+import BackcountryLogo from '../src/logos/BackCountry-logo.svg'
+import { useNavigate } from 'react-router-dom';
 
 function App() {
+  const navigate = useNavigate();
   const env = process.env.NODE_ENV;
   return (
-    <div className="m-[40px]">
-      <p>
-        <Link to="sso">Log in with SSO</Link>
-        <p>Environment: {env}</p>
-      </p>
+    <div>
+      <section className="bg-gray-50 dark:bg-gray-900">
```

Comment:
remove all dark: related css.
It is not relevant to our dev effort.

Reply:
addressed